docs/reference/architecture/system-contracts.md

```diff
@@ -85,38 +85,10 @@ It is called only once on the diamond constructor and is not saved in the diamon
 
 Implementation detail - function returns a magic value just like it is designed in [EIP-1271](https://eips.ethereum.org/EIPS/eip-1271), but the magic value is 32 bytes in size.
 
-### DiamondCutFacet
```

Comment:
Why is this removed?

Reply:
Some Facets were removed as part of Boojum upgrade : https://github.com/matter-labs/era-contracts/commit/5ca368a4377568b003c585dcd105265fd693f1d7#diff-f49fd64b738e337bc94e5cd566add8765352f0cd2c222ee4257958e7eed8b6c7